README.md

```diff
@@ -217,16 +217,19 @@ yarn install
 popd
 ```
 
-Then similarly with `matrix-react-sdk`:
+For `matrix-react-sdk` we are using a subtree and our own fork:
+When you clone tchap-web-v4, you should be able to see the repo in `linked-dependencies/matrix-react-sdk`.
 
-```bash
-git clone https://github.com/matrix-org/matrix-react-sdk.git
-pushd matrix-react-sdk
-yarn link
-yarn link matrix-js-sdk
-yarn install
-popd
+Don't forget to run a first yarn install inside `linked-dependencies/matrix-react-sdk`. Our `package.json`
+will be using this directory as local dependencies for `matrix-react-sdk`
+
+You can make changes to the subtree repo as if it was code from the parent repo. 
+But you have to make sure to also update the fork using those commands :
```

Comment:
if I understand subtrees right (heh), you only need to do this if you want to contribute to the upstream repo. In that case you need to push to a branch in your fork, and open a PR between branch-in-your-form and upstream.
Otherwise I don't see the point on maintaining the fork ? Two sources of truth would cause problems, no?